search/ranker.cpp

```diff
@@ -509,8 +510,8 @@ void Ranker::UpdateResults(bool lastUpdate)
 
   // Emit feature results.
   size_t count = m_emitter.GetResults().GetCount();
-  size_t i;
-  for (i = 0; i < m_tentativeResults.size(); ++i)
+  size_t i = 0;
+  for (; i < m_tentativeResults.size(); ++i)
   {
     if (!lastUpdate && i >= kBatchSize && !m_params.m_viewportSearch)
       break;
```

Comment:
[optional, may be other PR]. Next line is calling a function 'BailIfCancelled' where 'Bail' is looks like a misprint of 'Fail'. May be then rename BailIfCancelled to FailIfCancelled?